app/views/RegisterView.tsx

```diff
@@ -172,60 +172,64 @@ class RegisterView extends React.Component<IProps, any> {
 			return null;
 		}
 		try {
-			return Object.keys(this.parsedCustomFields).map((key, index, array) => {
-				if (this.parsedCustomFields[key].type === 'select') {
-					const options = this.parsedCustomFields[key].options.map((option: string) => ({ label: option, value: option }));
-					return (
-						<RNPickerSelect
-							key={key}
-							items={options}
-							onValueChange={value => {
-								const newValue: { [key: string]: string | number } = {};
-								newValue[key] = value;
-								this.setState({ customFields: { ...customFields, ...newValue } });
-							}}
-							value={customFields[key]}>
+			return (
+				<>
+					{Object.keys(this.parsedCustomFields).map((key, index, array) => {
+						if (this.parsedCustomFields[key].type === 'select') {
+							const options = this.parsedCustomFields[key].options.map((option: string) => ({ label: option, value: option }));
+							return (
+								<RNPickerSelect
+									key={key}
+									items={options}
+									onValueChange={value => {
+										const newValue: { [key: string]: string | number } = {};
+										newValue[key] = value;
+										this.setState({ customFields: { ...customFields, ...newValue } });
+									}}
+									value={customFields[key]}>
+									<TextInput
+										inputRef={(e: any) => {
+											// @ts-ignore
```

Comment:
Can we try the `@ts-ignore` on this file ?

Reply:
I tried, but I didn't find a solution, maybe later we can evaluate this again.